src/lib/Libutil/execvnode_seq_util.c

```diff
@@ -554,6 +574,9 @@ get_execvnodes_count(char *str)
 	if (str == NULL)
 		return 0;
 
+	if (str[0] == '(')
```

Comment:
Can you give an example of the str this function expects? not sure why it is returning the number of occurrences as 1 when it sees open braces.

Reply:
An execvnode_seq looks like N#(execvnode){0-N-1}.  This looked for the N at the front and failed with a normal exec_vnode.  A single exec_vnode will start with a '('.  I'll update the doxygen comment.